kafka-commons/build.gradle

```diff
@@ -3,7 +3,7 @@ ext {
 }
 
 dependencies {
-    compile "mesosphere:dcos-commons:0.4.37-SNAPSHOT"
+    compile "mesosphere:dcos-commons:0.5.0-SNAPSHOT"
```

Comment:
Latest is 0.7.0-SNAPSHOT now, might be worth catching up, if it's not a lot of work.  If it is, forget it.